ls1intum/Artemis
# `General`: Reduce loading time of the first page visit

This PR reduces the main bundle size from ~2MB to ~1.2MB and will significantly improve the first usage experience for slow Internet connections (unfortunately still quite common in Germany)
see https://bamboo.ase.in.tum.de/browse/ARTEMIS-WEBAPP3370-16/artifact/shared/webpack-bundle-report/stats.html




### Checklist
#### General



#### Client

### Motivation and Context
Reduce bundle size of the client application by using more lazy modules in Angular 

### Description


### Steps for Testing
1. Click through the most important pages and check that page loading and page styling works correctly
2. Test in particular the pages with discussions (Metis), teams and exercises for the students perspective
3. Also test some basic interaction on these pages to make sure nothing is broken
4. Ideally open the Console to check for typescript errors

### Review Progress





#### Code Review
#### Manual Tests
